src/Employment/WorkContract_v0.1.py

```diff
@@ -0,0 +1,339 @@
+    )
+    probation: str = Field(
+        ...,
+        title="Probation",
+        max_length=250,
+        description="The terms for the period of conditional employment",
+        examples=[
+            "The probationary period lasts 2 months from the employment start date"
+        ],
+    )
+
+
+class PaymentGrounds(str, Enum):
+    MONTHLY = "monthly"
+    HOURLY = "hourly"
+
+
+class Compensation(CamelCaseModel):
+    payment_grounds: PaymentGrounds = Field(
+        ...,
+        title="Payment Grounds",
+        description="The grounds for paying the salary",
+        examples=[PaymentGrounds.MONTHLY, PaymentGrounds.HOURLY],
+    )
+    salary: float = Field(
+        ...,
+        title="Salary",
+        description="The amount of salary in euros",
+        examples=[2500],
+    )
+    bonuses: Optional[str] = Field(
+        None,
+        title="Bonuses",
+        description="The details of the salary bonuses",
+        examples=["No additional bonuses"],
+        max_length=250,
+    )
+    payment_time: str = Field(
+        ...,
+        title="Payment Time",
+        description="The conditions for paying the salary",
+        examples=["The 15th of every month"],
+        max_length=250,
+    )
+
+
+class BenefitType(str, Enum):
+    PART_OF_SALARY = "part of salary"
+    ADDITION_TO_SALARY = "addition to salary"
+
+
+class Benefit(CamelCaseModel):
+    benefit: Optional[str] = Field(
+        None,
+        title="Benefit",
+        max_length=250,
+        description="The name of the benefit received as a compensation",
+        examples=["Lunch benefit"],
+    )
+    benefit_type: Optional[BenefitType] = Field(
+        None,
+        title="Benefit Type",
+        description="The type of the taxable benefit",
+        examples=[BenefitType.PART_OF_SALARY, BenefitType.ADDITION_TO_SALARY],
+    )
+    taxable_value: Optional[float] = Field(
+        None,
+        title="Taxable Value",
+        description="The taxable value for the compensation per month",
+        examples=[200],
+    )
+
+
+class DeterminationOfHoliday(str, Enum):
+    ANNUAL_HOLIDAY_ACT = "annual holiday act"
+    COLLECTIVE_AGREEMENT = "collective agreement"
+
+
+class Holidays(CamelCaseModel):
+    paid_holiday: bool = Field(
+        ...,
+        title="Paid Holiday",
+        description="Indicates if the contract includes paid holiday",
+        examples=[True],
+    )
+    number_of_holidays: Optional[int] = Field(
+        None,
+        title="Number Of Holidays",
+        description="The terms used for the paid holidays",
+        examples=[5],
+    )
+    determination_of_holiday: Optional[DeterminationOfHoliday] = Field(
+        None,
+        title="Determination Of Holiday",
+        description="The determination of the holiday",
+        examples=["The holiday is determined by the employer"],
+    )
+
+
+class Term(CamelCaseModel):
+    term_description: Optional[str] = Field(
```

Comment:
It makes no sense for this to be optional. You would then have a list of `Term`s, where some have a `term_description` (only thing in it) set to `null`. That's not desirable, rather the item should be left out in that case. Checked this with Henna and updated spreadsheet, so please update this to be a required field.